.azure_pipelines/build-pipeline.yml

```diff
@@ -6,7 +6,7 @@
 
 stages:
 - stage: 'Build'
-  displayName: 'Build PyTorch'
+  displayName: 'Build'
```

Comment:
We should probably leave the azure pipelines out of this since we don't have visibility into their testing. Are these even used at this time?

@mstfbl what are the plans for these pipelines?

Reply:
yea that sounds wise I'll update to exclude these